cmd/nerdctl/container/restart.go

```diff
@@ -63,10 +66,10 @@ func restartAction(cmd *cobra.Command, args []string) error {
 			if found.MatchCount > 1 {
 				return fmt.Errorf("multiple IDs found with provided prefix: %s", found.Req)
 			}
-			if err := stopContainer(ctx, found.Container, timeout); err != nil {
+			if err := action.StopContainer(ctx, found.Container, timeout); err != nil {
 				return err
 			}
-			if err := startContainer(ctx, found.Container, false, client); err != nil {
+			if err := action.StartContainer(ctx, found.Container, false, client); err != nil {
```

Comment:
Similar to other `container(s)` lifecycle related func. Putting them in the `pkg(/containerutil)` side might be better.